docs/trace_connectors_user_guide.md

```diff
@@ -85,6 +85,9 @@ Refer to the [Merge 1 User Guide](https://s3.amazonaws.com/Merge1Public/User%20G
 4. Globanet pushes the data to Relativity Fileshare's `{FILESHARE_WORKSPACE_ROOT}\DataTransfer\Import\Globanet_Data\{DATA_SOURCE_ARTIFACT_ID}\Drop` folder
 5. Globanet pushes any logs to Relativity Fileshare's `{FILESHARE_WORKSPACE_ROOT}\DataTransfer\Import\Globanet_Data\{DATA_SOURCE_ARTIFACT_ID}\Logs` folder
 
+## Setting up Trace Shipper Service
+If VPN/direct option for data delivery is not available you can configure Trace Shipper Service.  Refer to the [Trace Shipper Service Documentation](Trace Shipper Service.md) for instructions on how to install and configure Trace Shipper Service. Contact support@relativity.com if you need assitance.

```

Comment:
@brianheinz It seems like we are calling out Shipper as a backup option to VPN/direct option. Is that correct? I'm guessing the priority is Direct, Shipper, then VPN.

Reply:
Spoke with Misha. This makes sense since Shipper is in BETA.